### ISSUE TITLE
BUG: I'm getting Error: Cannot find module '/home/runner/github-readme-activity-graph-1/dist/main.js' in replit.com

### ISSUE THREAD

**Issue by hareesh-r** [Ashutosh00710/github-readme-activity-graph#175]
**Describe the bug**
I'm getting Error: Cannot find module '/home/runner/github-readme-activity-graph-1/dist/main.js' in replit.com

**To Reproduce**
Followed the steps in the Readme.md

https://replit.com/@hareeshr/github-readme-activity-graph-1

**Expected behavior**
The link should be generated

**Screenshots**
![image](https://user-images.githubusercontent.com/53457844/206205544-281c8c58-115b-4bb5-9643-dbf1f8060a09.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser - Brave Browser
 - Version : [Version 1.46.134 Chromium: 108.0.5359.94 (Official Build) (64-bit)](https://brave.com/latest/)

**Additional context**
```  bash replit.sh

up to date, audited 881 packages in 6s

51 packages are looking for funding
  run `npm fund` for details

7 vulnerabilities (1 low, 2 moderate, 4 high)

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

> github-readme-activity-graph@1.0.0 start
> node dist/main.js

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/home/runner/github-readme-activity-graph-1/dist/main.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
exit status 1```

**Comment by nishith-p-shetty** [comment]
i am going to fix this
edit a file replit.sh

**Comment by Ashutosh00710** [comment]
fixed by #179
